docs/reference/filters.md

```diff
@@ -82,23 +82,28 @@ The `Filter` has 4 properties:
 
 The available operators are:
 
-| Value   | Meaning               | Additional information                                    |
-|---------|-----------------------|-----------------------------------------------------------|
-| eq      | equal                 |                                                           |
-| not_eq  | different             |                                                           |
-| gt      | greater than          | against textual values, the alphabetical ordering is used |
-| gte     | greater than or equal | against textual values, the alphabetical ordering is used |
-| lt      | lower than            | against textual values, the alphabetical ordering is used |
-| lte     | lower than or equal   | against textual values, the alphabetical ordering is used |
-| nil     | empty / no value      | `nil` do not require anything inside `values`             |
-| not_nil | non-empty / any value | `not_nil` do not require anything inside `values`         |
-
-In addition, there are operators:
-
-- `starts_with` / `not_starts_with` / `ends_with` / `not_ends_with` / `contains` / `not contains`, available for searching in short string fields (name, value, title, etc.),
-- `search`, available in short string and text fields.
-
-There is a small difference between `search` and `contains`. `search` finds any occurrence of specified words, regardless of order, while "contains" specifically looks for the exact sequence of words you provide.
+| Value           | Meaning               | Available for         |
+|-----------------|-----------------------|-----------------------|
+| eq              | equal                 |                       |
+| not_eq          | different             |                       |
+| gt              | greater than          | numbers               |
+| gte             | greater than or equal | numbers               |
+| lt              | lower than            | numbers               |
+| lte             | lower than or equal   | numbers               |
+| nil             | empty / no value      |                       |
+| not_nil         | non-empty / any value |                       |
+| starts_with     | starts with           | short string          |
+| not_starts_with | doesn't start with    | short string          |
+| ends_with       | ends with             | short string          |
+| not_ends_with   | doesn't end with      | short string          |
+| contains        | contains              | short string          |
+| not_contains    | doesn't contain       | short string          |
+| search          | have occurences       | short and long string |
+
+Precisions:
+- The operators `nil` and `not_nil` don't require anything inside `values`.
+- For the comparison operators, it's the alphabetical ordering that is used against textual values.
```

Comment:
above we say it's only available for numbers, so usable in textual values as well ?

Reply:
in the UI it's only available for number... I'm gonna remove this line to avoid confusion